test/common/testlib.py

```diff
@@ -939,6 +939,10 @@ def become_superuser(
 
         self.open_superuser_dialog()
 
+        # In (open)SUSE images, superuser access always requires the root password
+        if "suse" in self.machine.image:
+            user = "root"
+
```

Comment:
This should only affect the implicit default, not if `user` is explicitly specified on calling the method. This should extend the `{user or 'admin'}` fallback below. I.e. something like
```
if user is None:
    user = "root" if "suse" in self.machine.image else "admin"
```
(do keep the comment, though), and dropping the `or` fallback below.